fjord/base/middleware.py

```diff
@@ -163,4 +163,5 @@ def process_request(self, request):
 
         request.path_info = '/' + prefixer.shortened_path
         request.locale = prefixer.locale
-        tower.activate(prefixer.locale)
+        translation.activate(prefixer.locale)
+        install_gettext()
```

Comment:
I do know that without it, templates blow up because they don't have gettext. This will change with django-jinja though so it is kind of temporary.